src/seedu/addressbook/AddressBook.java

```diff
@@ -1187,4 +1209,27 @@ private static String removePrefixSign(String s, String sign) {
 	    
 	    return results;
 	}
+		
+	private static class PersonComparator implements Comparator<String[]> {
+		@Override
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.